src/ts_type.rs

```diff
@@ -337,171 +340,206 @@ impl TsTypeDef {
 
       match &type_element {
         TsMethodSignature(ts_method_sig) => {
-          let mut params = vec![];
+          if let Some(prop_js_doc) =
+            js_doc_for_range(parsed_source, &ts_method_sig.range())
+          {
+            let mut params = vec![];
```

Comment:
Looks like this could be pre-allocated? `let mut params = Vec::with_capacity(ts_method_sig.params.len());`?